packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -307,3 +307,29 @@ impl PersistedEntityType {
         &self.metadata
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct EntityTypeRootedSubgraph {
```

Comment:
Perhaps this is just a lack of rebase, but this definitely shouldn't be here

Reply:
I have not rebased the branch in a while. With the latest rebase this is gone again.

Reply:
Thought so. Thanks!